internal/scene/play.go

```diff
@@ -40,6 +41,8 @@ func NewPlay(c controller) *Play {
 	p.registerAction(ebiten.KeyB, action.ToggleBoundingBoxes)
 	p.registerAction(ebiten.KeySpace, action.Pause)
 	p.registerAction(ebiten.KeyT, action.ToggleTextures)
+	p.registerAction(ebiten.KeyC, action.ToggleCollisions)
+	p.registerAction(ebiten.KeyD, action.ToggleCollisionBoxes)
```

Comment:
Is not the bounding box the collision box?

Reply:
It is different. The BB is useful if we want not to draw things but their area box. It doesn't mean they can have collision. Imagine we have entities for pollution on the map, they can't collide with anything but we still can draw their BB.